sql/moz-fx-data-shared-prod/adjust/adjust_derived/view.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE VIEW
+    `moz-fx-data-shared-prod.analysis.adjust_derived_v1`
+AS
+SELECT
+    *
+FROM `moz-fx-data-shared-prod.analysis.mhirose_adjust_derived_raw`
```

Comment:
this doesn't seem to match the destination table name?

Reply:
You're right! I hadn't checked this part. I'm not sure a view is even necessary.